drainer/server.go

```diff
@@ -269,6 +273,7 @@ func (s *Server) Start() error {
 	go func() {
 		for err := range errc {
 			log.Error("send heart failed", zap.Error(err))
+			s.reportError(err)
```

Comment:
it seems no need to quit once it failed to update the statue here, the replication can still work.
how about only failed and quit it failed to update status for a long time?